app/design/adminhtml/Magento/backend/Magento_Config/web/css/source/_module.less

```diff
@@ -147,4 +147,9 @@
             display: none;
         }
     }
+
+    select:disabled,
+    input:disabled {
+        .__form-control-pattern__disabled();
+    }
```

Comment:
These changes apply also over all other input and select elements in Admin Panel, even those which already have disabled/[disabled]/.disabled styling applied.

Reply:
Hi @ptylek :
I changed it to .admin__control-table-wrapper select:disabled and .admin__control-table-wrapper input:disabled. Please check it again. Thanks!